rudder-tests/schema.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python3
+
+
+# Sample file, to try to build a basic schema for platform inputs
+
+import os
+import json
+import re
+import sys
+import traceback
+from subprocess import Popen, PIPE
+from jsonschema import validate, draft7_format_checker, Draft7Validator, RefResolver
+import importlib
+
+###################
+
+def run_scenario(name, datastate, json_file=None):
```

Comment:
I don't understand, this file is called schema.py and contains a function to run a scenario ?